qiskit/pulse/instructions/acquire.py

```diff
@@ -13,7 +13,7 @@
 # that they have been altered from the originals.
 
 """The Acquire instruction is used to trigger the qubit measurement unit and provide
-some metadata for the acquisition process; for example, where to store classified readout data.
+some metadata for the acquisition process. for example, where to store classified readout data.
```

Comment:
```suggestion
some metadata for the acquisition process, for example, where to store classified readout data.
```
"where to store classified readout data" isn't a whole sentence